build_npm.ts

```diff
@@ -4,11 +4,24 @@ import { build, emptyDir } from "https://deno.land/x/dnt/mod.ts";
 await emptyDir("./npm");
 
 await build({
+  typeCheck: false,
   entryPoints: ["./src/mod.ts"],
   outDir: "./npm",
   shims: {
     // see JS docs for overview and more options
     deno: true,
+    // custom shims
+    custom: [{
+      package: {
+        name: "node-fetch",
+        // Please see: https://www.npmjs.com/package/node-fetch-commonjs
+        version: "2.6.7", 
+      },
+      globalNames: [{
+        name: "fetch",
+        exportName: "default"
+      }]
+    }],
   },
```

Comment:
Let's add an explicit compilation target here, like 
```suggestion
  },
    compilerOptions: {
    target: "ES2020",
  },
```

This makes the emitted package work on node v14 (though it won't work on node v12).
In combination with the above change, can we also bump up the minimum `engines` node version in `package.json` to match the minimum node version that supports ES2020 (not sure exactly which version that is - at least node v14, but may be even a node v13 something, though I wouldn't spend too much time trying to figure that out, I doubt any sane person is using the odd, unstable node versions).